projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -781,4 +787,5 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   enablePasswordsCannotMatchInPasswordUpdateForm: false,
   allPageMetaResolversEnabledInCsr: false,
   useExtendedMediaComponentConfiguration: false,
+  realTimeStockDispaly: false,
```

Comment:
NITPICK: can we make this to `showRealTimeStockInPDP` similar to how it was done for promotions `showPromotionsInPDP` ?

Reply:
done